config/config.go

```diff
@@ -516,9 +516,13 @@ func checkTimeInterval(r *Route, timeIntervals map[string]struct{}) error {
 			return err
 		}
 	}
-	if len(r.MuteTimeIntervals) == 0 {
-		return nil
+
+	for _, mt := range r.ActiveTimeIntervals {
```

Comment:
we need also a test in `*Config.UnmarshalYAML()` to prevent active time intervals on the root route.

Reply:
Agreed I will include the test, thanks

Reply:
The test was added